index.js

```diff
@@ -86,14 +93,11 @@ function init() {
 	return ret;
 }
 
-defineProps(chalk, init());
+defineProps(Chalk.prototype, init());
 
-chalk.styles = ansiStyles;
-chalk.hasColor = hasAnsi;
-chalk.stripColor = stripAnsi;
-chalk.supportsColor = supportsColor;
+Chalk.prototype.styles = ansiStyles;
```

Comment:
yep, I'll get that fixed. thanks poking me was getting was almost forgetting about this :)